x/config/socks5.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"net/url"
+
+	"github.com/Jigsaw-Code/outline-sdk/transport"
+	"github.com/Jigsaw-Code/outline-sdk/transport/socks5"
+)
+
+func newSOCKS5StreamDialerFromURL(innerDialer transport.StreamDialer, configURL *url.URL) (transport.StreamDialer, error) {
```

Comment:
Please update the Go doc with the new parameter. It doesn't show the credentials currently: https://pkg.go.dev/github.com/Jigsaw-Code/outline-sdk/x/config

Reply:
@fortuna good catch! it slipped my mind. I updated the doc to include authentication params (`UserInfo`) in the URL.